tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -54,6 +54,11 @@ final class ClosureLinkerBackend(
 
   val symbolRequirements: SymbolRequirement = emitter.symbolRequirements
 
+  private val needsIIFEWrapper = moduleKind match {
+    case ModuleKind.NoModule       => true
+    case ModuleKind.CommonJSModule => false
+  }
```

Comment:
Hum ... any reason you're not using the same test as [`Emitter.needsIIFEWrapper`](https://github.com/scala-js/scala-js/blob/cc6117a5ccf226bc4dfde00f12d64183446a93c1/tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/Emitter.scala#L63-L76)? Or even *reuse* `Emitter.needsIIFEWrapper`?